### PR TITLE
tests: ad add column tests

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

Adds a few basic `add column tests` all using the context menu for now. 

# Testing

Check them out. 

# Documentation

No.

### CODE REVIEW

File: tests/streamlit_ui_tests/add_column_tests.spec.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const mito = await getMitoFrameWithTypeCSV(page);
+    await createNewColumn(page, mito, 3, 'Column4');
+});
+
+test('Add new column to start using context menu', async ({ page }) => {
+    const mito = await getMitoFrameWithTypeCSV(page);
+    await createNewColumn(page, mito, 0, 'Column4');
+});
+
+test('Add new column to middle using context menu', async ({ page }) => {
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/add_column_tests.spec.ts:15]: Could chop this for the next test

File: tests/streamlit_ui_tests/add_column_tests.spec.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { FrameLocator, Page, expect, test } from '@playwright/test';
+import { getMitoFrameWithTypeCSV, createNewColumn, clickButtonAndAwaitResponse } from './utils';
+
+
+test('Add new column to end using context menu', async ({ page }) => {
+    const mito = await getMitoFrameWithTypeCSV(page);
+    await createNewColumn(page, mito, 3, 'Column4');
```
Comment (naterush) [inline tests/streamlit_ui_tests/add_column_tests.spec.ts:7]: You're not actually checking the column exists after! There's utilities in the pivot PR that make this super easy.

Comment (naterush) [reply tests/streamlit_ui_tests/add_column_tests.spec.ts:7]: Applies to all of these tests

File: tests/streamlit_ui_tests/utils.ts [hunk 3]
```diff
@@ -71,9 +71,48 @@ export const closeTaskpane = async (mito: FrameLocator): Promise<void> => {
 export const getColumnHeaderContainer = async (mito: FrameLocator, columnName: string): Promise<Locator> => {
     return mito.locator('.endo-column-header-container').locator('div').filter({ hasText: columnName }).first();
 }
+
+export const getColumnHeaderContainerAtIndex = async (mito: FrameLocator, index: number): Promise<Locator> => {
+    return mito.locator('.endo-column-header-container').nth(index);
+}
+
+export const renameColumnByIndex = async (page: Page, mito: FrameLocator, index: number, newName: string): Promise<void> => {
```
Comment (naterush) [inline tests/streamlit_ui_tests/utils.ts:79]: renameColumnAtIndex perhaps

**Comment by naterush** [comment]
(LGTM when tests pass)

**Pull request merged**